test/test_geometries/custom_lon_lat_point.hpp

```diff
@@ -72,14 +72,7 @@ struct access<rw_lon_lat_point<CoordinateType, CoordinateSystem>, Dimension>
     void set(rw_lon_lat_point<CoordinateType, CoordinateSystem>& p,
              CoordinateType const& value)
     {
-        if (Dimension == 0)
-        {
-            p.longitude = value;
-        }
-        else
-        {
-            p.latitude = value;
-        }
+        ( Dimension == 0 ? p.longitude : p.latitude ) = value;
```

Comment:
Never seen this before. Does this work everywhere?
I prefer the old version, but than with BOOST_GEOMETRY_CONDITION as done everywhere else